### PR TITLE
[Merged by Bors] - chore: bump fluvio version to 0.10.12-dev-1

### PR DESCRIPTION
**Pull request by digikata**

### CODE REVIEW

**Review by morenol — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ serde_json = "1.0.79"
 node-bindgen = "5.1.0"
 flate2 = "1.0.24"
 fluvio-future = { version = "0.5.1", features = ["tls", "task", "io"] }
-fluvio = { features = ["admin"], git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.11" }
+fluvio = { features = ["admin"], git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.12-dev-1" }
 fluvio-spu-schema = { git = "https://github.com/infinyon/fluvio.git", tag = "v0.10.11" }
```
Comment (morenol) [inline Cargo.toml:25]: we need to update this one too

Comment (digikata) [reply Cargo.toml]: good catch, picked up in latest push

**Comment by digikata** [comment]
note: semver audit issue see https://github.com/infinyon/fluvio-client-node/issues/336

**Pull request closed without merging**